release-notes/releases-index.json

```diff
@@ -0,0 +1,44 @@
+{
+    "core-releases-index": [
+        {
+            "channel-version": "2.1",
+            "latest": "2.1.4",
+            "release-date":"2018-09-11",
+            "lts": true,
+            "eol": false,
+            "eol-date": null,
+            "releases.json": "https://github.com/dotnet/core/blob/master/release-notes/2.1/releases.json",
+            "lifecycle-policy": "https://www.microsoft.com/net/support/policy"
```

Comment:
I'd make "lifecycle-policy" a top-level property and include it per releases within the target releases.json file.

Reply:
It's already in the target releases.json files. The question is, is duplicating this top level information in the index useful?

Reply:
Primary point is not duplicating it in every array entry in this file. We can likely get away w/o it being in this file at all. There are many links that we could include here. Only one of them is required.